package.json

```diff
@@ -35,7 +35,7 @@
     "serve-built-storybook": "npx http-server docs",
     "verify-node-version": "chmod +x ./scripts/verify-node-version.sh && ./scripts/verify-node-version.sh",
     "preversion": "node ./src/generateVersion.js",
-    "version": "npm run preversion && npm run verify-node-version && npm run build-storybook && git add -u ./docs && git add ./docs/* && git add ./src/version.ts && npm run compile",
+    "version": "npm run preversion && npm run verify-node-version && npm run build-storybook && git add -u ./docs && git add ./docs/* && git add ./src/version.ts && git add package.json",
```

Comment:
I've adjusted this slightly for the purpose of the automation, since when we're incrementing the version  for the new commit we do not need the compiled code. The compile step will be called on its own in the publish to NPM workflow.

Reply:
I see that the previous and edited version of this "version" command are explictly calling `npm run preversion` ...

I don't think we need to call pre and post commands explicitly. [npm docs on topic here](https://docs.npmjs.com/cli/v10/using-npm/scripts#pre--post-scripts)

Reply:
actually, I'm wondering if we want to refactor this:
```
    "preversion": "node ./src/generateVersion.js",
    "version": "npm run preversion && npm run verify-node-version && npm run build-storybook && git add -u ./docs && git add ./docs/* && git add ./src/version.ts",
```

to something like this:
```
    "version": "node ./src/generateVersion.js && npm run verify-node-version",
     "postversion": "npm run build-storybook && git add -u :/docs && git add ./src/version.ts",
```

Reply:
I've removed the unneeded preversion call, but as per our sync discussion we can't do the docs generation and git staging in our postversion since the commit is already created and tagged by this point.